### DISCUSSION TITLE
wechat-adjust-position 没有奏效

### DISCUSSION THREAD

**Discussion by hecool108** [remaxjs/remax#1875] [General]
hi 各位辛苦了：

我发现为TextArea设置wechat-adjust-position这个属性为false是没有效果的
```javascript
<Textarea 
    wechat-adjust-position={false}
    wechat-hold-keyboard={true}>
</Textarea>
```
用同样为wechat准备的属性wechat-hold-keyboard是没问的。

remax版本：2.15.13
微信版本： 8.0.18
运行系统: iOS 15.0
基础库版本：2.23.3[684]

**Comment by watsonhaw5566** [comment]
` <Textarea 
adjustPosition={"false"} 
holdKeyboard={true}> 测试一下
</Textarea>`
 引用的时候 remax 已经有使用类型提示了，根据 remax 的来 不用加入而 不用加入 wechat 前缀的

属性是生效的了，当你定义为 false 的时候需要自行处理当前的页面。
https://developers.weixin.qq.com/community/develop/doc/000a4268fdc1388d3768e204d56c00
